src/containerapp/azext_containerapp/_params.py

```diff
@@ -394,6 +394,10 @@ def load_arguments(self, _):
         c.argument('consumer_secret_setting_name', options_list=['--consumer-secret-name', '--secret-name'], help='The consumer secret name that contains the app secret.')
         c.argument('provider_name', required=True, help='The name of the custom OpenID Connect provider.')
         c.argument('openid_configuration', help='The endpoint that contains all the configuration endpoints for the provider.')
+        c.argument('token_store', options_list=['--token-store'], arg_type=get_three_state_flag(), help='true if the token store feature is enabled for the current app; otherwise, false.')
```

Comment:
```suggestion
c.argument('token_store', options_list=['--token-store'], arg_type=get_three_state_flag(), help='Boolean indicating if the environment is enabled to have token store.')
```

Reply:
`arg_type=get_three_state_flag()` with append `Allowed values: false, true.` in the argument property.
![image](https://github.com/Azure/azure-cli-extensions/assets/46143499/0c6cdab5-af46-4673-b6c1-d86cf99d4001)